packages/connect-node/src/connect-http-transport.ts

```diff
@@ -109,7 +107,9 @@ export interface ConnectHttpTransportOptions {
   /**
    * Options for the http request.
    */
-  httpOptions?: http.RequestOptions | https.RequestOptions;
+  httpOptions?:
+    | Omit<http.RequestOptions, "signal">
+    | Omit<https.RequestOptions, "signal">;
```

Comment:
We should not accept an AbortSignal in the request options because we already accept an AbortSignal in CallOptions.